features/index/filters.feature

```diff
@@ -35,6 +35,20 @@ Feature: Index Filtering
     And I press "Filter"
     Then I should see current filter "title_cont" equal to "<script>alert('hax')</script>" with label "Title contains"
 
+  Scenario: Daterange Filters Uses Locale Format
+    Given an index configuration of:
+    """
+      ActiveAdmin.register Post
+    """
+    When I set my locale to "de"
+    And I am on the index page for posts
+    And I fill in "q[published_date_gteq]" with "20.10.1999"
+    And I press "Filtern"
+    Then the "q[published_date_gteq]" field should contain "20.10.1999"
+    When I set my locale to "en"
+    And I press "Filtern"
```

Comment:
```suggestion
    And I press "Filter"
```

Reply:
No, it's still a German page at that point. I am only doing that, because the I18n locale is not reset otherwise...

Reply:
(the  `When I set my locale to "en"` does not reload the page)

Reply:
Yes, that makes sense. At the time I thought this may have been one possible issue so wanted to call it out just in case. Thank you.